core/src/gl/uniformEntries.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <string>
+
+namespace Tangram {
+
+namespace UniformEntries {
```

Comment:
I'm fine to change the name for `Uniform`. See the other comment for the need of `UniformEntry`.